src/app/components/profile-membership/profile-membership.component.ts

```diff
@@ -1,26 +1,32 @@
 import { Component, OnInit } from '@angular/core';
 import { UserService } from 'src/app/services/user-service/user.service';
 import { User } from 'src/app/models/user';
+import { Car } from 'src/app/models/car';
+import { CarService } from 'src/app/services/car-service/car.service';
 @Component({
   selector: 'app-profile-membership',
   templateUrl: './profile-membership.component.html',
   styleUrls: ['./profile-membership.component.css']
 })
 export class ProfileMembershipComponent implements OnInit {
-  profileObject : User;
-  currentUser: any = '';
-  isDriver: boolean;
-  active: boolean;
+  profileObject: User;
+  carObject: Car;
   success: string;
-  constructor(private userService: UserService) { }
+  possibleSeats: Array<Number>;
```

Comment:
Why is possible seats represented by an array?

Reply:
used an array so that the app can dynamically populate the options of the seatsAvailable select. The array is populated with a sequence of numbers up to the set number of seats in the car model.